megamek/src/megamek/common/loaders/BLKDropshipFile.java

```diff
@@ -45,6 +45,21 @@ public BLKDropshipFile(BuildingBlock bb) {
         dataFile = bb;
     }
 
+    @Override
+    public int defaultVGLFacing(int location, boolean rearFacing) {
```

Comment:
Two questions:

1) Is there a way to avoid this copy-paste across many files?
2) Will this for spheroid dropships?

Reply:
1. The loaders have a lot of repeated code. They really should be in a class hierarchy appropriate to the unit type rather than all inheriting directly from BlkFile. But that's way beyond the scope of this.
2. Yes. Rear side arcs are implemented as rear-mounted equipment on the wings.